main.py

```diff
@@ -0,0 +1,56 @@
+import os
+import argparse
+from WeatherDataParser import WeatherDataParser
+from WeatherDataCalculator import WeatherDataCalculator
+from WeatherReport import WeatherReport
+
+
+def main():
+    arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument(
+        "dir",
+        help="Directory which contains Weather Files.", nargs='?')
+    arg_parser.add_argument(
+        "-e",
+        help="For a given year display the highest temperature and day, lowest temperature and day, most humid day and humidity.")
+    arg_parser.add_argument(
+        "-a",
+        help="For a given month display the average highest temperature, average lowest temperature, average mean humidity.")
+    arg_parser.add_argument(
+        "-c",
+        help="For a given month draw two horizontal bar charts on the console for the highest and lowest temperature on each day. Highest in red and lowest in blue.")
+    args = arg_parser.parse_args()
+
+    if args.dir and os.path.exists(args.dir):
+        murree_weather_data = {}
```

Comment:
Weather data should not be specified to a city. Lets avoid this kind of variable name. 
Also, please avoid using data types in variable names.